core-web/libs/edit-content/src/lib/utils/functions.util.ts

```diff
@@ -44,3 +51,19 @@ export const getSingleSelectableFieldOptions = (
 
     return result;
 };
+
+// This function is used to cast the value to a correct type for the Angular Form
+export const getFinalCastedValue = (value: string | null, field: DotCMSContentTypeField) => {
```

Comment:
Use Document This! extension.

Reply:
That extension does not like arrow functions.